src/__tests__/cogniteClient.integration.ts

```diff
@@ -16,3 +17,18 @@ describe('createClientWithApiKey - integration', () => {
     ).rejects.toThrowErrorMatchingInlineSnapshot(`"Unauthorized | code: 401"`);
   });
 });
+
+describe('http methods - integration', () => {
+  let client: CogniteClient;
+  beforeAll(async () => {
+    client = setupLoggedInClient();
+  });
+  test('post method', async () => {
+    const assets = [{ name: 'First asset' }, { name: 'Second asset' }];
+    const response = await client.post(
+      '/api/v1/projects/cognitesdk-js/assets',
+      { data: { items: assets } }
+    );
```

Comment:
have only for post? I mean, it's fine, but I though you were going to make it for all of them :)

Reply:
Made one for get as well, as put and delete is not supported in v1 yet.